lib/if.c

```diff
@@ -1193,7 +1193,10 @@ DEFPY_YANG_NOSH (interface,
 	 * VRF.
 	 */
 	VRF_GET_ID(vrf_id, vrf_name, false);
-	ifp = if_lookup_by_name_all_vrf(ifname);
+	if (vrf_get_backend() == VRF_BACKEND_VRF_LITE)
+		ifp = if_lookup_by_name_all_vrf(ifname);
+	else
+		ifp = if_lookup_by_name(ifname, vrf_id);
```

Comment:
This won't work if you have an interface in a VRF, but you don't specify the VRF name in the command.
I think we should do something like:
```
if (vrf_name)
    ifp = if_lookup_by_name(ifname, vrf_id);
else
    ifp = if_lookup_by_name_all_vrf(ifname);
```
regardless of the vrf backend.

Reply:
proposal is ok for me.